mod-vendors/schemas/contact.json

```diff
@@ -85,6 +84,14 @@
         "$ref": "url.json"
       }
     },
+    "categories": {
+      "id": "contact_category",
+      "description": "The list of contact categories for this vendor contact person",
```

Comment:
Probably worth noting that it's really an array of UUIDs pointing to category records.  You probably also want to include the UUID pattern on the string